paddlecloud/paddlejob/views.py

```diff
@@ -96,10 +96,7 @@ def post(self, request, format=None):
         # jobPackage validation: startwith /pfs
         # NOTE: job packages are uploaded to /pfs/[dc]/home/[user]/jobs/[jobname]
         job_name = obj.get("name", "paddle-cluster-job")
-        if settings.STORAGE_MODE == "CEPHFS":
-            package_in_pod = os.path.join("/pfs/%s/home/%s"%(dc, username), "jobs", job_name)
-        elif settings.STORAGE_MODE == "HDFS":
```

Comment:
If we remove current method to deal with HDFS, shall we put a design doc and re-design ways to be compatible to it?

In another PR will be OK~